lib/api/include/irods/get_grid_configuration_value.h

```diff
@@ -0,0 +1,34 @@
+#ifndef IRODS_GET_GRID_CONFIGURATION_VALUE_H
+#define IRODS_GET_GRID_CONFIGURATION_VALUE_H
+
+/// \file
+
+struct RcComm;
+struct GridConfigurationInput;
+struct GridConfigurationOutput;
```

Comment:
perhaps, the actual header that defines these types should be included.

this PR currently requires that the user of the API manually include `grid_configuration_types.h`.

very inconvenient.